### ISSUE TITLE
微调中途报错Exception: Current loss scale already at minimum - cannot decrease scale anymore. Exiting run.

### ISSUE THREAD

**Issue by cat-sun** [FlagAlpha/Llama2-Chinese#66]

**Comment by cat-sun** [comment]
tr_loss_step = self.training_step(model, inputs)  File "/root/ddalgo/LLaMA/finetune/LLaMA2/src/train_bash.py", line 10, in main
    tr_loss_step = self.training_step(model, inputs)
        train_result = trainer.train()tr_loss_step = self.training_step(model, inputs)
    tr_loss_step = self.training_step(model, inputs)
    tr_loss_step = self.training_step(model, inputs)
        self.accelerator.backward(loss)self.accelerator.backward(loss)
        return inner_training_loop(self.accelerator.backward(loss)
    self.accelerator.backward(loss)
    self.accelerator.backward(loss)
        self.accelerator.backward(loss)self.accelerator.backward(loss)
        self.deepspeed_engine_wrapped.backward(loss, **kwargs)self.deepspeed_engine_wrapped.backward(loss, **kwargs)
    self.deepspeed_engine_wrapped.backward(loss, **kwargs)
tr_loss_step = self.training_step(model, inputs)
    self.deepspeed_engine_wrapped.backward(loss, **kwargs)
self.deepspeed_engine_wrapped.backward(loss, **kwargs)
        self.deepspeed_engine_wrapped.backward(loss, **kwargs)self.deepspeed_engine_wrapped.backward(loss, **kwargs)
    self.accelerator.backward(loss)
        self.optimizer.step()self.deepspeed_engine_wrapped.backward(loss, **kwargs)
        self.loss_scaler.update_scale(has_overflow)self.loss_scaler.update_scale(has_overflow)
  File "/root/miniconda3/envs/llama2_etuning/lib/python3.10/site-packages/deepspeed/runtime/fp16/loss_scaler.py", line 175, in update_scale
  File "/root/miniconda3/envs/llama2_etuning/lib/python3.10/site-packages/deepspeed/runtime/fp16/loss_scaler.py", line 175, in update_scale
    self.loss_scaler.update_scale(has_overflow)
  File "/root/miniconda3/envs/llama2_etuning/lib/python3.10/site-packages/deepspeed/runtime/fp16/loss_scaler.py", line 175, in update_scale
self.loss_scaler.update_scale(has_overflow)
self.loss_scaler.update_scale(has_overflow)
  File "/root/miniconda3/envs/llama2_etuning/lib/python3.10/site-packages/deepspeed/runtime/fp16/loss_scaler.py", line 175, in update_scale
Exception  File "/root/miniconda3/envs/llama2_etuning/lib/python3.10/site-packages/deepspeed/runtime/fp16/loss_scaler.py", line 175, in update_scale
Exception: : Current loss scale already at minimum - cannot decrease scale anymore. Exiting run.Current loss scale already at minimum - cannot decrease scale anymore. Exiting run.
Exception:     Current loss scale already at minimum - cannot decrease scale anymore. Exiting run.raise Exception(
    self.loss_scaler.update_scale(has_overflow)
Exception  File "/root/miniconda3/envs/llama2_etuning/lib/python3.10/site-packages/deepspeed/runtime/fp16/loss_scaler.py", line 175, in update_scale
Exception:     : Current loss scale already at minimum - cannot decrease scale anymore. Exiting run.self.loss_scaler.update_scale(has_overflow)    Current loss scale already at minimum - cannot decrease scale anymore. Exiting run.
  File "/root/miniconda3/envs/llama2_etuning/lib/python3.10/site-packages/deepspeed/runtime/fp16/loss_scaler.py", line 175, in update_scale
Exception: Current loss scale already at minimum - cannot decrease scale anymore. Exiting run.
Exception: Current loss scale already at minimum - cannot decrease scale anymore. Exiting run.
    self.loss_scaler.update_scale(has_overflow)
  File "/root/miniconda3/envs/llama2_etuning/lib/python3.10/site-packages/deepspeed/runtime/fp16/loss_scaler.py", line 175, in update_scale
Exception: Current loss scale already at minimum - cannot decrease scale anymore. Exiting run.
[2023-08-01 23:45:39,147] [ERROR] [launch.py:321:sigkill_handler] ['/root/miniconda3/envs/llama2_etuning/bin/python', '-u', 'src/train_bash.py', '--local_rank=7', '--deepspeed', 'deepspeed.json', '--stage', 'sft', '--prompt_template', 'llama2', '--model_name_or_path', '/root/ddalgo/models/Llama2-Chinese-7b-Chat-Lora-Pretrained', '--do_train', '--dataset', 'train_sft', '--finetuning_type', 'lora', '--output_dir', '/root/ddalgo/LLaMA/finetune/LLaMA2/sft_chinese_llama2', '--overwrite_cache', '--per_device_train_batch_size', '16', '--gradient_accumulation_steps', '8', '--lr_scheduler_type', 'cosine', '--logging_steps', '10', '--save_steps', '100', '--learning_rate', '5e-5', '--num_train_epochs', '3.0', '--plot_loss', '--fp16'] exits with return code = 1
[2023-08-02 09:43:11,959] [INFO] [runner.py:555:main] cmd = /root/miniconda3/envs/llama2_etuning/bin/python -u -m deepspeed.launcher.launch --world_info=eyJsb2NhbGhvc3QiOiBbMCwgMSwgMiwgMywgNCwgNSwgNiwgN119 --master_addr=127.0.0.1 --master_port=29501 --enable_each_rank_log=None src/train_bash.py --deepspeed deepspeed.json --stage sft --prompt_template llama2 --model_name_or_path /root/ddalgo/models/Llama2-Chinese-7b-Chat-Lora-Pretrained --do_train --dataset train_sft --finetuning_type lora --output_dir /root/ddalgo/LLaMA/finetune/LLaMA2/sft_chinese_llama2 --overwrite_cache --per_device_train_batch_size 16 --gradient_accumulation_steps 8 --lr_scheduler_type cosine --logging_steps 10 --save_steps 100 --learning_rate 5e-5 --num_train_epochs 3.0 --plot_loss
prediction_loss_only=False,
prediction_loss_only=False,
prediction_loss_only=False,
prediction_loss_only=False,
prediction_loss_only=False,
prediction_loss_only=False,
prediction_loss_only=False,
prediction_loss_only=False,
[2023-08-02 09:54:49,934] [INFO] [config.py:964:print]   dynamic_loss_scale_args ...... None
[2023-08-02 09:54:49,934] [INFO] [config.py:964:print]   loss_scale ................... 0
        "loss_scale": 0,
        "loss_scale_window": 1000,
        "min_loss_scale": 1
    tr_loss_step = self.training_step(model, inputs)
    tr_loss_step = self.training_step(model, inputs)
    loss = self.compute_loss(model, inputs)
  File "/root/miniconda3/envs/llama2_etuning/lib/python3.10/site-packages/transformers/trainer.py", line 2679, in compute_loss
    loss = self.compute_loss(model, inputs)
  File "/root/miniconda3/envs/llama2_etuning/lib/python3.10/site-packages/transformers/trainer.py", line 2679, in compute_loss
    loss = self.module(*inputs, **kwargs)
    loss = self.module(*inputs, **kwargs)
    tr_loss_step = self.training_step(model, inputs)
    loss = self.compute_loss(model, inputs)
  File "/root/miniconda3/envs/llama2_etuning/lib/python3.10/site-packages/transformers/trainer.py", line 2679, in compute_loss
    loss = self.module(*inputs, **kwargs)
    tr_loss_step = self.training_step(model, inputs)
    loss = self.compute_loss(model, inputs)
  File "/root/miniconda3/envs/llama2_etuning/lib/python3.10/site-packages/transformers/trainer.py", line 2679, in compute_loss
    loss = self.module(*inputs, **kwargs)
    tr_loss_step = self.training_step(model, inputs)
    loss = self.compute_loss(model, inputs)
  File "/root/miniconda3/envs/llama2_etuning/lib/python3.10/site-packages/transformers/trainer.py", line 2679, in compute_loss
    tr_loss_step = self.training_step(model, inputs)
    loss = self.module(*inputs, **kwargs)
loss = self.compute_loss(model, inputs)
  File "/root/miniconda3/envs/llama2_etuning/lib/python3.10/site-packages/transformers/trainer.py", line 2679, in compute_loss
    loss = self.module(*inputs, **kwargs)
    tr_loss_step = self.training_step(model, inputs)
    tr_loss_step = self.training_step(model, inputs)
    loss = self.compute_loss(model, inputs)
  File "/root/miniconda3/envs/llama2_etuning/lib/python3.10/site-packages/transformers/trainer.py", line 2679, in compute_loss
    loss = self.compute_loss(model, inputs)
outputs = run_function(*args)  File "/root/miniconda3/envs/llama2_etuning/lib/python3.10/site-packages/transformers/trainer.py", line 2679, in compute_loss
    loss = self.module(*inputs, **kwargs)
        return forward_call(*args, **kwargs)loss = self.module(*inputs, **kwargs)

**Comment by DericZhao** [comment]
解决了么，是不是v100卡的问题
